PixiEditor/Models/UserPreferences/IPreferences.cs

```diff
@@ -7,26 +7,84 @@ public interface IPreferences
     {
         public static IPreferences Current => ViewModelMain.Current.Preferences;
 
+        /// <summary>
+        /// Saves the preferences to be stored permanently.
+        /// </summary>
         public void Save();
 
-        public void AddCallback(string setting, Action<object> action);
+        /// <summary>
+        /// Add's a callback that will be executed when the setting called <paramref name="name"/> changes.
```

Comment:
there shouldn't be an apostrophe in "add's", same check the rest of the file for the same mistake because there are like 8 of them (when you have nothing to say about the code so you become a grammar nazi instead)